skopt/optimizer/optimizer.py

```diff
@@ -176,7 +183,95 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                              "'sampling', got {0}".format(acq_optimizer))
         self.acq_optimizer = acq_optimizer
 
-    def ask(self):
+
+
+
+    def copy(self):
+        """Creates a shallow copy of optimizer."""
+        optimizer = Optimizer(
+            dimensions=self.space.dimensions,
+            base_estimator=self.base_estimator,
+            n_random_starts=self._n_random_starts,
+            acq_func=self.acq_func,
+            acq_optimizer=self.acq_optimizer,
+            random_state=copy.copy(self.rng),
+            acq_func_kwargs=self.acq_func_kwargs,
+            acq_optimizer_kwargs=self.acq_optimizer_kwargs,
+        )
+
+        if hasattr(self, "gains_"):
+            optimizer.gains_ = np.copy(self.gains_)
+
+        if len(self.Xi) > 0:
+            optimizer.tell(self.Xi, self.yi)
+
+        return optimizer
+
+
+
+    def ask(self, n_points=None, parallel_strategy="cl_min"):
+        """Query point or multiple points to evaluate.
+
+        * `n_points` [int or None]:
+            Number of function evaluations that can be done in parallel.
+            If the value is None, a single point to evaluate is returned.
+            Otherwise a list of points to evaluate is returned of size n_jobs.
+
+        * `parallel_strategy` [string, default=`"cl_min"`]:
```

Comment:
`parallel_strategy` to just strategy? 

I understand that you mean parallel as in "parallel in evaluation", but it can also mean that the strategy is parallel in nature but in reality it is sequential